collector/dispersion.go

```diff
@@ -0,0 +1,194 @@
+}
+
+func newDispersionReportDumpTask(pathToDispersionExecutable string) collectorTask {
+	return &dispersionReportDumpTask{
+		pathToDispersionExecutable: pathToDispersionExecutable,
+		containerCopiesExpected: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_container_copies_expected",
+				"Expected container copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		containerCopiesFound: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_container_copies_found",
+				"Found container copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		containerCopiesMissing: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_container_copies_missing",
+				"Missing container copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		containerOverlapping: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_container_overlapping",
+				"Expected container copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		objectCopiesExpected: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_object_copies_expected",
+				"Expected object copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		objectCopiesFound: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_object_copies_found",
+				"Found object copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		objectCopiesMissing: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_object_copies_missing",
+				"Missing object copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+		objectOverlapping: typedDesc{
+			desc: prometheus.NewDesc(
+				"swift_dispersion_object_overlapping",
+				"Expected object copies reported by the swift-dispersion-report tool.",
+				nil, nil),
+			valueType: prometheus.GaugeValue,
+		},
+	}
+}
+
+// dispersionReportDumpTask implements the collector.collectorTask interface.
+func (t *dispersionReportDumpTask) describeMetrics(ch chan<- *prometheus.Desc) {
+	t.containerCopiesExpected.describe(ch)
+	t.containerCopiesFound.describe(ch)
+	t.containerCopiesMissing.describe(ch)
+	t.containerOverlapping.describe(ch)
+	t.objectCopiesExpected.describe(ch)
+	t.objectCopiesFound.describe(ch)
+	t.objectCopiesMissing.describe(ch)
+	t.objectOverlapping.describe(ch)
+}
+
+// dispersionReportDumpTask implements the collector.collectorTask interface.
+func (t *dispersionReportDumpTask) collectMetrics(ch chan<- prometheus.Metric, exitCodeTypedDesc typedDesc) {
+	exitCode := 0
+	cmdArg := "--dump-json"
+	out, err := exec.Command(t.pathToDispersionExecutable, cmdArg).Output()
```

Comment:
```suggestion
	out, err := exec.Command(t.pathToDispersionExecutable, cmdArg).CombinedOutput()
```
so that we also see logs from stderr in the error message.